### PR TITLE
Migrate from virtualenv to pipenv

### PR DESCRIPTION
**Pull request by nre-ableton**
Related to https://github.com/AbletonAppDev/devtools/issues/1322, ping @AbletonDevTools/gotham-city

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -23,7 +23,10 @@ runTheBuilds.runDevToolsProject(
       },
       hadolint: {
         docker.image('hadolint/hadolint:v1.6.6').inside("-v ${pwd()}:/ws") {
-          sh 'hadolint /ws/Dockerfile'
+          // DL3013: Pin versions in pip
+          // We are only using pip to install pipenv, and we laways want the latest and
```
Comment (ala-ableton) [inline Jenkinsfile]: Typo: laways -> always

**Comment by mst-ableton** [comment]
@nre-ableton can you add a small description to this PR?

**Comment by nre-ableton** [comment]
@mst-ableton oh sorry, I was waiting for it to pass CI first before my obligatory ping.

**Review by mst-ableton — APPROVED** [review]
lgtm!

**Pull request merged**